src/zimg/colorspace/gamma.cpp

```diff
@@ -250,6 +260,10 @@ TransferFunction select_transfer_function(TransferCharacteristics transfer, doub
 	func.to_gamma_scale = 1.0f;
 
 	switch (transfer) {
+	case TransferCharacteristics::LOG_100:
+		func.to_linear = scene_referred ? log100_inverse_oetf : rec_1886_eotf;
```

Comment:
I don't think Rec.1886 is correct for LOG-encoded images.

Reply:
hm you're right, the colours do not appear correctly
do you think the log transfers could be describing EOTFs instead?

Reply:
Could be that there is no OETF/EOTF distinction for log.